src/components/WwwFrame/TheHeader.vue

```diff
@@ -669,6 +669,11 @@ export default {
 	},
 	methods: {
 		toggleLendMenu(immediate = false) {
+			if (!this.isLendMenuVisible) {
+				// If menu was previously hidden, run menu onLoad
+				this.$refs?.lendMenu?.onLoad?.();
+			}
```

Comment:
Moving this seems risky as it previously depended on updated state from that whole `immediate` block...

Reply:
Ok I think all of those conditions below are to handle various states that dictate IF the menu should call the load function. Bypassing the checks looks like it could affect behavior on touch devices and change the auto close behavior as well...

Reply:
I did test all browser sizes, but I will find an alternate solution.